services/search.service.js

```diff
@@ -52,7 +52,17 @@ function executeQuery(model, queryArray){
                 break;
         }
     }
-    return query.lean().exec()
+
+    if(sort == 'desc'){
+        query.sort('-'+sort_by);
+    }
+    else if(sort == 'asc'){
+        query.sort(sort_by);
+    }
+    return query.lean()
+        .limit(limit)
+        .skip(limit * page)
```

Comment:
Perhaps add a comment here to clarify why this makes sense? Probs just one liner

Reply:
I see how you're using limit and skip for pagination, but where are we changing the page value? If we need 10 pages, are we submitting 10 queries, each with a different page value?

Reply:
Both are parameters passed through the query in the url.
Often times for various dashboards that have pagination you have a dropdown that lets you denote how many results you want at a time (25,50,100,1000 etc) and that works with the page so that on the second page you see results 26-50.
But yes if you want 10 pages that's how you would do it, I think that's how it's usually done as well so you don't get everything sent to your browser.